src/index.html

```diff
@@ -11,6 +11,18 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" method="POST">
+      <input
+        class="search-bar search-bar--big"
+        type="text" data-qa="big"
+        placeholder="Try “Los Angeles”"
+      >
```

Comment:
Still not fixed from the previous review
```suggestion
      <input
        class="search-bar search-bar--big"
        type="text"
        data-qa="big"
        placeholder="Try “Los Angeles”"
      >
```